Formula/mingw-w64.rb

```diff
@@ -123,6 +123,7 @@ def install
       mkdir "#{buildpath}/gcc/build-#{target_arch}" do
         system "../configure", *gcc_args
         system "make", "all-gcc"
+        system "make", "check-gcc"
```

Comment:
Please don't `make check-gcc` as part of the build, it runs the full testsuite and can take several hours